backend/UnicornRewards.API/Program.cs

```diff
@@ -17,8 +19,11 @@
 });
 builder.Services.AddMicrosoftIdentityWebApiAuthentication(builder.Configuration, "AzureAd");
 builder.Services.AddControllers();
+builder.Services.AddScoped<IContactService<string>, ContactService>();
+builder.Services.AddCors();
 var app = builder.Build();
 app.UseSerilogRequestLogging();
+app.UseCors(c => c.AllowAnyHeader().AllowAnyMethod().AllowAnyOrigin());

```

Comment:
No need to 'allow everything' with Cors, start out by allowing only the things you need to allow (Headers and Methods) the origins can be left as AllowAnyOrigin.
Try to follow the principle of least privilege always

Reply:
Removed permission, defined "allow" only to GET and PUT methods, change updated on "fixes to contact list functionality" commit